utils/src/main/java/uk/gov/di/authentication/utils/lambda/EmailCheckResultWriterHandler.java

```diff
@@ -59,7 +68,11 @@ public Void emailCheckResultWriterHandler(SQSEvent event) {
                 LOG.info(
                         "Message for email check reference {} written to database",
                         emailCheckResult.referenceNumber());
+                var timeFromInitialRequest =
+                        now().getTime() - emailCheckResult.timeOfInitialRequest();
 
+                cloudwatchMetricsService.putEmbeddedValue(
+                        "timeFromInitialRequest", timeFromInitialRequest, Map.of());
```

Comment:
At the top of `java/uk/gov/di/authentication/shared/services/CloudwatchMetricsService.java` I notice we have a set of imports to some defined metrics.

Can we do the same here, rather than create the string `"timeFromInitialRequest"`?

Looking at the pattern already set up here, I wonder if we'd be better off naming this metric something like `EmailCheckDuration`

Reply:
Good points to raise. Agreed. I would probably suggest adding a couple of dimensions. Looking at the other implementations in `CloudwatchMetricsService.java` I would suggest the following could be helpful dimensions to add
```
ACCOUNT.getValue(),
accountState.name(),
ENVIRONMENT.getValue(),
configurationService.getEnvironment(),
IS_TEST.getValue(),
Boolean.toString(isTestJourney),
```

The ones about the client are kind of irrelevant for the knowing how long the email checker took.

Reply:
Alright, agreed
Adding them now

Reply:
Matt, may I suggest just adding the environment, email and email check status to the cloudwatch Logs?
The other information can be retrieved only from a `userSession`, which we do not have in this handler and I am quite reluctant to add it to the `EmailCheckResultWriterHandler` as it is in the shared `utils` project

Reply:
Hey, makes sense. 
- Environment 👍, go for it
- Email, probably not, as it's PII.